cli/test/test_run_parser.py

```diff
@@ -387,10 +387,10 @@ def get_parser_output(
     """Generate the parser output objects for the tests given input variables."""
     return ParserOutput(
         document_id="sdf",
-        document_metadata=BackendDocument.parse_obj(document_metadata),
+        document_metadata=BackendDocument.model_validate(document_metadata),
         document_name="sdf",
         document_description="sdf",
-        document_source_url=source_url,
+        document_source_url=source_url,  # type: ignore
```

Comment:
Is this needing an ignore because of its type definition above, You might want the optional type hint instead:
`Optional[str]`

If its something else and not an obvious fix it would be worth adding a bit more context. I think you can ignore specific issues like so:
`type: ignore[name-of-error]`

Reply:
I'll update this now my dad!

Reply:
Haha 👴

Reply:
Haha ffs